src/index.js

```diff
@@ -26,6 +27,7 @@ const updateStatus = (index) => {
 };
 
 const display = () => {
+  console.log('HI');
```

Comment:
> * [ ] Please remove this `console.log` as it's not a best practice to have it in your code and sometimes, it could log sensitive data like user passwords. 
Please also remove it in other places where it's been used in your code.